Client/YooChooseNotifier.php

```diff
@@ -222,14 +224,18 @@ public function hideLocation($locationId, $isChild = false)
      */
     public function unhideLocation($locationId)
     {
-        $location = $this->locationService->loadLocation($locationId);
-        $children = $this->locationService->loadLocationChildren($location)->locations;
+        if (!$this->hasCredentials()) {
+            return;
+        }
+
+        $location = $this->repositoryHelper->loadLocation($locationId);
+        $children = $this->repositoryHelper->loadLocationChildren($location)->locations;
 
         foreach ($children as $child) {
             $this->unhideLocation($child->id);
```

Comment:
no way to send these events in bulk? might be quite a bit of children to send events for

Reply:
@adamwojs do you have some info how many events we can send to this endpoint? https://github.com/ezsystems/EzSystemsRecommendationBundle/blob/f368e89b201383a926791c3458eb32aa70d462a0/Client/YooChooseNotifier.php#L349

Reply:
@damianz5 I will check it 😉